controllers/openstackcluster_controller.go

```diff
@@ -258,7 +258,7 @@ func deleteBastion(scope scope.Scope, cluster *clusterv1.Cluster, openStackClust
 	return nil
 }
 
-func reconcileNormal(scope scope.Scope, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster) (ctrl.Result, error) {
+func reconcileNormal(scope scope.Scope, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster) (ctrl.Result, error) { //nolint:unparam
```

Comment:
um... from code all 3 params are used , not sure why need such update to make CI happy

Reply:
Ah now I see... the issue is that the returned `ctrl.Result` is always empty == `nil`.

Reply:
Yeah. I thought about this one a bit. We could remove the ctrl.Result return from this function, but:
* That would make it different to all other reconcile methods (and therefore Surprising)
* It might encourage lazy programming when we inevitably want to add it back